### ISSUE TITLE
Could you provide the pre-trained model weights used in the paper?

### ISSUE THREAD

**Issue by FantasticGNU** [Tokichan/CSAD#2]
Hi, thanks for sharing the code! I'm trying to reproduce the results from the paper using the provided training script and generated semantic pseudo-label maps, but my results are much lower than expected (I got 90.29). Could you provide the pre-trained model weights used in the paper? Thanks!

**Comment by Tokichan** [comment]
Hi, I found a few bugs in the released code, and a fix will be released
soon along with the pretrained weight,
thank you!

FantasticGNU ***@***.***> 於 2024年9月5日 週四 下午4:23寫道：
